src/wf_utils/labels_tw.config

```
@@ -11,9 +14,19 @@ process {
   withLabel: midcpu { cpus = 15 }
   withLabel: highcpu { cpus = 30 }
   
-  withLabel: lowmem { memory = { get_memory( 20.GB * task.attempt ) } }
-  withLabel: midmem { memory = { get_memory( 50.GB * task.attempt ) } }
-  withLabel: highmem { memory = { get_memory( 100.GB * task.attempt ) } }
+  withLabel: lowmem { 
+                      memory = { get_memory( 20.GB * task.attempt ) } 
+                      disk = { 50.GB * task.attempt } 
+                    }
+  withLabel: midmem { 
+                      memory = { get_memory( 50.GB * task.attempt ) }
+                      disk = { 100.GB * task.attempt } 
+                    }
+  withLabel: highmem {
+                      memory = { get_memory( 100.GB * task.attempt ) }
+                      disk = { 200.GB * task.attempt } 
+                    }
```

Undo weird indentation ;)

```suggestion
  withLabel: lowmem { 
    memory = { get_memory( 20.GB * task.attempt ) } 
    disk = { 50.GB * task.attempt } 
  }
  withLabel: midmem { 
    memory = { get_memory( 50.GB * task.attempt ) }
    disk = { 100.GB * task.attempt } 
  }
  withLabel: highmem {
    memory = { get_memory( 100.GB * task.attempt ) }
    disk = { 200.GB * task.attempt } 
  }
```